### ISSUE TITLE
Question about compatibility, more testers

### ISSUE THREAD

**Issue by scripting** [scripting/wordlandSupport#28]
This is primarily for @jeherve --

### The current compatibility statement is incorrect.

On the WordLand about page, the [limit on compatibility](https://this.how/wordland/#1733149380000) is stated thus:

> At this time WordLand only works with sites hosted on wordpress.com.

I understand that's not true. What is the correct way to state the limit?

Something in this form perhaps:

> WordLand works with sites hosted on wordpress.com and self-hosted sites that have the xxx plugin installed with the xxx setting true (the default).

Also is "self-hosted" the right terminology?

### More testers sought

Also I'm posting on my blog that we're looking for more testers, "with good experience with bug reporting and using WordPress." If you can recommend friends or colleagues, please ask them to fill out the form. Esp people from inside the community. I want this product to really fit in the WordPress culture as much as possible, given that I am not rooted in that culture, help is appreciated.

**Comment by jeherve** [comment]
> WordLand works with sites hosted on wordpress.com and self-hosted sites that have the xxx plugin installed with the xxx setting true (the default).

Yup, that would work. The plugin being Jetpack and the feature being the "JSON API module".

> is "self-hosted" the right terminology?

Yes, that's what's commonly used in the WordPress community.

**Comment by scripting** [comment]
@jeherve -- thanks! 

i've [updated](https://this.how/wordland/#1733149380000) the docs.

**Comment by scotthansonde** [comment]
> > WordLand works with sites hosted on wordpress.com and self-hosted sites that have the xxx plugin installed with the xxx setting true (the default).

I was able to set up a new self-hosted WordPress site and got WordLand to publish to it. Here are my loose notes (I had a couple false starts connecting Jetpack to wordpress.com)

- Install "WordPress with Mysql" on [coolify](https://coolify.io/)
- Fill out the welcome information for a new site
- WordPress 6.7.1 
- Install Pulitzer theme
- Install Jetpack ("Jetpack – WP Security, Backup, Speed, & Growth")
- "Activate Jetpack in one click"
	- It knew my email address, so it connected right to my wordpress.com account
	- On Jetpack "Manage My Sites", then "Approve"
	- Add a site, Learn how to add a site
	- In Wordpress Jetpack, connect, complete setup
	- Jetpack is successfully connected
	- Ignore all the paid plans...
	- The new install shows up as one of my "Sites" on wordpress.com
- In WordLand my new install shows up as an option
- Wrote a post, published it, and it worked!

My first post is at https://wordland-test.shanson.de/2024/12/23/hello-world-2/

**Comment by scripting** [comment]
@scotthansonde -- this is great! :smile:

**Comment by scripting** [comment]
a question about what's enabled by @scotthansonde's new server.

if scott could give me a site on his serve, can i tell wordpress.com to include it in the list of sites it returns when i sign on with wordland?

if this can work, then wordpress is much more than i thought it was -- it's basically a distributed network of sites.

and no rush on any of this. tis the season ho ho ho! :-)

![image](https://github.com/user-attachments/assets/9bef4e2d-3a1b-49f5-a92e-f765a16f8568)

**Comment by jeherve** [comment]
> if scott could give me a site on his serve, can i tell wordpress.com to include it in the list of sites it returns when i sign on with wordland?

If the Jetpack plugin is installed on a site, all registered users on that site can connect their own account to their own WordPress.com account. That means that you could indeed publish posts on Scott's site from WordLand, if you had an author account on that site. 

There are no limits to the number of sites you can connect to, so the list of sites displayed in WordLand really only depends on the number of sites (WordPress.com or self-hosted with Jetpack) where you have an account.

**Comment by scripting** [comment]
a question has come up with the release of bingeworthy, which like wordland, uses the wpidentity api to connect to wordpress for identity now, but as mentioned elsewhere for writing eventually. the question of course has also recently come up for wordland too. 

the question is -- can you use an account on another wordpress server or does it have to be wordpress.com.

For background...

https://github.com/scripting/Scripting-News/issues/316#issuecomment-2583290746

**Comment by jeherve** [comment]
> can you use an account on another wordpress server or does it have to be wordpress.com.

wpIdentity only allows you to authenticate to WordPress.com. It does not support any other service. 

Once you're authenticated, you can make API requests (e.g. publish posts) to any site that ships with [the WordPress.com REST API endpoints](https://developer.wordpress.com/docs/api/). All sites hosted on WordPress.com include those endpoints, as well as sites hosted elsewhere and where the Jetpack plugin is installed.
